auth0_flutter/EXAMPLES.md

```diff
@@ -331,6 +331,9 @@ The Credentials Manager utility allows you to securely store and retrieve the us
 
 When the users open your app, check for valid credentials. If they exist, you can retrieve them and redirect the users to the app's main flow without any additional login steps.
 
+<details>
```

Comment:
This section is marked with the following:

> This feature is mobile-only; on web, the [SPA SDK](https://github.com/auth0/auth0-spa-js) used by auth0_flutter keeps its own cache.

Including documentation for web in it may make it confusing. Maybe we can document the web parts somewhere else?

Reply:
Plus we don't really have a Credentials Manager for the web platform.

Reply:
Yes that's true. I could just document them under it's own thing around getting/checking credentials on the web?

Reply:
Perfect, let's go with that 👍🏼

Reply:
Updated it 👍🏻